.github/workflows/actions.yml

```diff
@@ -24,6 +28,14 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Build
+        run: |
+          make
+          if [ $? -ne 0 ]; then
+            echo "Build failed"
+            exit 1
+          fi
+
```

Comment:
I'm almost certain that in the context of the `run` in github actions, failure is automatically reported when a command exits with a nonzero status, but we may change that later to avoid potential breakage since I'm not 100% certain about it.